runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java

```diff
@@ -220,6 +234,48 @@ else if (dailyReset)
 		}
 	}
 
+	private void checkFlax(boolean dailyReset)
+	{
+		if (client.getVar(Varbits.DIARY_KANDARIN_EASY) == 1)
+		{
+			if (client.getVar(Varbits.DAILY_FLAX_STATE) == 0)
+			{
+				sendChatMessage(FLAX_MESSAGE);
+			}
+			else if (dailyReset)
+			{
+				sendChatMessage(FLAX_MESSAGE + RELOG_MESSAGE);
+			}
+		}
+	}
+
+	private void checkBonemeal(boolean dailyReset)
+	{
+		if (client.getVar(Varbits.DIARY_MORYTANIA_MEDIUM) == 1)
+		{
+			int collected = client.getVar(Varbits.DAILY_BONEMEAL_STATE);
+			//The varbit tracks how much bonemeal has been redeemed from Robin
```

Comment:
Comments on how the varbit work should probably be on the definition, not on the usage of it.